tools/spectrum-scale-driver-snap.sh

```diff
@@ -88,25 +88,25 @@ PRODUCT_NAME="ibm-spectrum-scale-csi"
 echo "Collecting \"$PRODUCT_NAME\" logs..."
 echo "The log files will be saved in the folder [$logdir]"
 
-csi_spectrum_scale_attacher_log_name=${logdir}/csi-spectrum-scale-attacher.log
-csi_spectrum_scale_provisioner_log_name=${logdir}/csi-spectrum-scale-provisioner.log
+csi_spectrum_scale_attacher_log_name=${logdir}/ibm-spectrum-scale-csi-attacher.log
+csi_spectrum_scale_provisioner_log_name=${logdir}/ibm-spectrum-scale-csi-provisioner.log
 
-describe_all_per_label=${logdir}/csi-spectrum-scale-describe-all-by-label
-get_all_per_label=${logdir}/csi-spectrum-scale-get-all-by-label
-get_configmap=${logdir}/csi-spectrum-scale-configmap
-get_k8snodes=${logdir}/csi-spectrum-scale-k8snodes
+describe_all_per_label=${logdir}/ibm-spectrum-scale-csi-describe-all-by-label
+get_all_per_label=${logdir}/ibm-spectrum-scale-csi-get-all-by-label
+get_configmap=${logdir}/ibm-spectrum-scale-csi-configmap
+get_k8snodes=${logdir}/ibm-spectrum-scale-csi-k8snodes
 
-echo "$klog StatefulSet/csi-spectrum-scale-attacher"
-$klog StatefulSet/csi-spectrum-scale-attacher > ${csi_spectrum_scale_attacher_log_name} 2>&1 || :
-echo "$klog StatefulSet/csi-spectrum-scale-provisioner"
-$klog StatefulSet/csi-spectrum-scale-provisioner > ${csi_spectrum_scale_provisioner_log_name} 2>&1 || :
+echo "$klog StatefulSet/ibm-spectrum-scale-csi-attacher"
+$klog StatefulSet/ibm-spectrum-scale-csi-attacher > ${csi_spectrum_scale_attacher_log_name} 2>&1 || :
+echo "$klog StatefulSet/ibm-spectrum-scale-csi-provisioner"
+$klog StatefulSet/ibm-spectrum-scale-csi-provisioner > ${csi_spectrum_scale_provisioner_log_name} 2>&1 || :
 
```

Comment:
Should we collect the applied scc(csi-plugin-scc.yaml_template)details as part of log collection ?

Reply:
Added